mix.exs

```diff
@@ -22,7 +22,7 @@ defmodule PlugDatadogStats.Mixfile do
   defp deps do
     [
       {:ex_statsd, github: "PagerDuty/ex_statsd", ref: "a5c1aefd1d8d273e3910c2ae53c034669f792400"}, # Pulling in Cees' socket open fix
-      {:plug, "~> 1.3.2"},
+      {:plug, ">= 1.3.2 and < 1.5.0"},
```

Comment:
Is there any particular reason we need to prevent 1.5?

Reply:
Nope. I could switch this to something like `>= 1.3.2 and ~> 1.3` to prevent upgrades to Plug 2.0. Or just leave it at `>= 1.3.2`. Opinion on which to go with?

Reply:
I would prefer we use `1.3` and let it be optimistic.

Reply:
We want something that requires `>= 1.3.2` - we're explicitly guarding against vulnerable versions of Plug in the 1.3 series. The vulnerability was fixed in `1.3.2`.